### PR TITLE
Adds exactly one new hat.

### PR DESCRIPTION
**Pull request by thatguyfromlife**
Adds the nurses hat in, since it was just sitting in our code and someone asked for it.

Everything should be working codewise, as far as I can tell.

### CODE REVIEW

**Comment by Superhats** [comment]
The nurse hat is questionable at best, but I really don't think we need more alt titles. We already have plenty bloating the codebase.

**Review by BurpleBineapple — CHANGES_REQUESTED** [review]
The only one I could see working here is "Nurse", the others feel more akin to bloat unfortunately.

**Comment by Keekenox** [comment]
Now it's just a nurse hat for the nurse title. Add add add add.

**Comment by thatguyfromlife** [comment]
The axing of titles is done, it only took 15 minutes for checks to realize.

**Review by Superhats — APPROVED** [review]
Without the alt titles, it's got my approval.

File: maps/torch/job/medical_jobs.dm [hunk 1]
```diff
@@ -146,7 +146,6 @@
 	ideal_character_age = 20
 	alt_titles = list(
 		"Corpsman Trainee")
-
```
Comment (BurpleBineapple) [inline maps/torch/job/medical_jobs.dm:149]: I'm gonna be picky, but please bonk this one random space here.

**Pull request merged**